src/electron/views/screenshot/highlight-box.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import { BoundingRectangle } from 'electron/platform/android/scan-results';
+import { highlightBox, highlightBoxLabel } from 'electron/views/screenshot/highlight-box.scss';
+import * as React from 'react';
+import { CSSProperties } from 'react';
+
+export interface HighlightBoxDeps {}
+export interface HighlightBoxProps {
+    deps: HighlightBoxDeps;
+    boundingRectangle: BoundingRectangle;
+}
+
+export const HighlightBox = NamedFC<HighlightBoxProps>('HighlightBox', props => {
+    const { boundingRectangle } = props;
+
+    const width = boundingRectangle.right - boundingRectangle.left;
+    const height = boundingRectangle.bottom - boundingRectangle.top;
+
+    const boxStyles: CSSProperties = {
+        width: `${width}px`,
+        height: `${height}px`,
+        top: `${boundingRectangle.top}px`,
+        left: `${boundingRectangle.left}px`,
+    };
+
+    return (
+        <div className={highlightBox} style={boxStyles}>
```

Comment:
We should check with @ferBonnin about the desired screen reader announcement for these boxes; we'll probably want to either explicitly hide them until we implement interactivity, or give them a more descriptive accessible text than `!`

Reply:
It would be better to not announce it to ATs.